api/src/pcapi/routes/backoffice_v3/finance/blueprint.py

```diff
@@ -114,7 +122,7 @@ def cancel_finance_incident(finance_incident_id: int) -> utils.BackofficeRespons
         return render_finance_incident(incident)
 
     try:
-        _cancel_finance_incident(incident)
+        _cancel_finance_incident(incident, form.data["comment"])
```

Comment:
```suggestion
        _cancel_finance_incident(incident, form.comment.data)
```
par homogénéité avec le reste du code backoffice